Library/Formula/mgba.rb

```diff
@@ -0,0 +1,38 @@
+class Mgba < Formula
+  desc "Game Boy Advance emulator"
+  homepage "https://mgba.io/"
+  url "https://github.com/mgba-emu/mgba/archive/0.6.3.tar.gz"
+  sha256 "35c8906c01206adae714fee9c2bc39698c514a4adb32c9c4a5a3fa58b2d10f9a"
+  head "https://github.com/mgba-emu/mgba.git"
+
+  depends_on :ld64 # fixes a linktime error on Tiger
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libepoxy"
+  depends_on "libpng"
+  depends_on "libzip"
+  depends_on "sdl2"
+  depends_on "sqlite"
```

Comment:
The sqlite-using feature isn't exposed in the SDL version. Might be better to just disable it to reduce build times.